rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-recorder-6.4.0-release-notes.adoc

```diff
@@ -0,0 +1,47 @@
+= RPA Recorder 6.4.0 Release Notes
+
+RPA Recorder enables you to record a business process for automation.
+
+include::partial$release-notes/rn-last-version-message.adoc[] 
+
+== July 31, 2023
+
+// == What's New
+include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+1]
+
+* You can now use image-based mouse clicks to record mouse clicks on a UI element that otherwise could not be recorded properly.
+* Japanese localization during Web recording is fully supported.
+
+// == Deprecated
+
+The following settings are removed from the Settings section of RPA Recorder:
+
+* *Merge keystrokes*
+* *Verbosity level*
+
+// == Compatibility
+include::partial$release-notes/rn-headings.adoc[tag=compatibility,leveloffset=+1]
+
+[%header%autowidth.spread]
+|===
+|Application |Version
+|RPA Builder | 6.4.0 and later
```

Comment:
praise: that's very important, thanks! RPA Builder <6.4.0 won't work with the recordings made with RPA Recorder 6.4.0